KILTprotocol/mashnet-node
# refactor: spiritnet runtime chain name

## fixes KILTProtocol/ticket#1409
* Fixes issue of having the same chain runtime strings across all runtimes
* Renamed dev nodes to `KILT Local` from `KILT Collator Local Testnet`

## How to test:

I tested this thoroughly with [this branch](https://github.com/KILTprotocol/polkadot-apps/tree/fix-kilt-chains) in the Polkadot Apps which I will open a PR for once this PR is merged.

For every node in `--chain={dev-specs/kilt-parachain/kilt-westend.json|dev-specs/kilt-parachain/peregrine-kilt.json|spiritnet-dev}`
1. Spin up a node
```
cargo run --release -p kilt-parachain -- --runtime spiritnet --chain=dev-specs/kilt-parachain/kilt-westend.json --collator --tmp --ws-port=9944 --port 30444 --alice --ws-external --rpc-external --rpc-cors all --rpc-methods=unsafe --unsafe-rpc-external -- --chain=dev-specs/kilt-parachain/kusama-relay.json
```
2. Run Polkadot.js on [this branch](https://github.com/KILTprotocol/polkadot-apps/tree/fix-kilt-chains) and connect to the node
Should see teal color if not running `KILT Local`.

## Checklist:

  - [x] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [x] This PR does not introduce new custom types
